config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :account do
+    resources :users
```

Comment:
was it a part of your task?

CREATE NEW dashboard#index controller in namespace account and output just "Hello <first_name" of current user